cmd/operator/main.go

```diff
@@ -125,6 +126,7 @@ func init() {
 	flagset.StringVar(&cfg.PrometheusDefaultBaseImage, "prometheus-default-base-image", "quay.io/prometheus/prometheus", "Prometheus default base image")
 	flagset.StringVar(&cfg.ThanosDefaultBaseImage, "thanos-default-base-image", "improbable/thanos", "Thanos default base image")
 	flagset.Var(ns, "namespaces", "Namespaces to scope the interaction of the Prometheus Operator and the apiserver.")
+	flagset.Var(promNS, "prometheus-instance-namespaces", "Namespaces where Prometheus CRDs and corresponding Secrets,Configmaps and StatefulSets are watched/created.")
```

Comment:
I think I'd prefer it to be the other way around.

Reply:
see my comments in https://github.com/coreos/prometheus-operator/issues/2186

Reply:
Nit: `Secrets,Configmaps` -> `Secrets, ConfigMaps`